source/vir/src/prune.rs

```diff
@@ -380,6 +383,13 @@ fn traverse_reachable(ctxt: &Ctxt, state: &mut State) {
             if ctxt.assert_by_compute && crate::interpreter::is_sequence_fn(&f).is_some() {
                 reach_seq_funs(ctxt, state);
             }
+            // set operations may be invoked for opens_invariants checks
+            reach_function(ctxt, state, &fn_set_contains_name(&ctxt.vstd_crate_name));
+            reach_function(ctxt, state, &fn_set_empty_name(&ctxt.vstd_crate_name));
+            reach_function(ctxt, state, &fn_set_full_name(&ctxt.vstd_crate_name));
+            reach_function(ctxt, state, &fn_set_insert_name(&ctxt.vstd_crate_name));
+            reach_function(ctxt, state, &fn_set_remove_name(&ctxt.vstd_crate_name));
+            reach_function(ctxt, state, &fn_set_subset_of_name(&ctxt.vstd_crate_name));
```

Comment:
can you move these into the OpenInvariant case?

Reply:
Even if a function doesn't open invariants, it might need to do mask set checking (to ensure that its mask is larger than those of any functions that it calls).  I will put these into both the `OpenInvariant` and `Call` cases, though; that should suffice.

Reply:
oh, that's a good point. Call is so ubiquitous though that it'll hardly make a difference to move it.